### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV Medications Prescription Renewal

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#95295]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the delayed feedback!
- Really appreciate seeing products at this stage. Design work is great too, but it's helpful sometimes to start thinking about the problem space just from the user flow.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** I'd encourage you to think about what happens if a user renews or refills a medication on accident. Maybe they're planning to talk to their doctor first but accidentally hit the button, or they're making choices about which medications they want to spend money on, etc. Can they cancel their request, or is it going through no matter what once they hit the button?
   
   My motivation here is WCAG requirements [3.3.4](https://www.w3.org/WAI/WCAG22/Understanding/error-prevention-legal-financial-data.html) (required under Section 508) and the stronger version of it, [3.3.6](https://www.w3.org/WAI/WCAG22/Understanding/error-prevention-all.html) (not required under 508 but a great thing to aim for). Unless money is changing hands or a legal obligation is created this interaction probably doesn't fall under 3.3.4 --- but it definitely falls under 3.3.6, which would require that a user can either reverse, check, or confirm their choice.
   
   There are a variety of ways you could do this (cancel button, confirmation modal, etc) but they're probably going to depend a ton on how the API you're connecting to works.
- [ ] **Must:** Now is a great time to start thinking about confirmation messages after a renewal or refill request goes through --- in particular, where it's placed in relation to the button the user just clicked, and what a user is likely to do next. (I'm thinking: Is a user going to select all the medications they want to refill and then hit the button? Or are their separate buttons for each medication and a user may click through a sequence of them? Per the item above, what happens if a user needs to cancel the request) This will help determine whether or not keyboard focus will need to move after the interaction, and if so where it should go.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
